# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/uusi/hotfix-cabal.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- uusi-0.4.3.0/uusi.cabal.vanilla	2025-09-30 10:16:50.025855710 -0400
+++ uusi-0.4.3.0/uusi.cabal	2025-09-30 10:17:06.151365386 -0400
@@ -34,7 +34,7 @@
 common common-options
   build-depends:
     , base   >=4.8 && <5
-    , Cabal  ^>=3.2 || ^>=3.4 || ==3.6.3 || ^>= 3.8 || ^>= 3.10
+    , Cabal  ^>=3.2 || ^>=3.4 || ==3.6.3 || ^>= 3.8 || ^>= 3.10 || ^>=3.12
     , text
 
   ghc-options:
--- uusi-0.4.3.0/app/Main.hs.vanilla	2025-09-30 10:21:22.797168802 -0400
+++ uusi-0.4.3.0/app/Main.hs	2025-09-30 10:26:51.041869484 -0400
@@ -59,5 +59,5 @@
 findCabal :: IO FilePath
 findCabal =
   findPackageDesc "." >>= \case
-    Left err -> fail err
+    Left err -> fail $ show err
     Right x -> pure x
